docs/en/getting-started/example-datasets/wikistat.md

```diff
@@ -4,15 +4,15 @@ sidebar_label: WikiStat
 
```

Comment:
PR updates ./docs/en/**. I think it's wrong. It should be in ./docs/ru/. Apparently it happens because ./docs/ru/../wikistat.md is a softlink to the english document